### DISCUSSION TITLE
How to hide column with autogenerated value

### DISCUSSION THREAD

**Discussion by micebrain** [nocodb/nocodb#724] [Q&A] [answered]
Hi,
I have a postgresql database with identity column. Is there a way to disable this column from data entry?

Thanks!

**Comment by wingkwong** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/35857179/139634163-ac92196d-222d-4cd5-88a0-8242061556f4.png)

You can hide the column by clicking ``Fields`` and unticking the columns you want to hide.

**Reply by micebrain** [reply]
Got it. Thanks!
